.idea/.name

```diff
@@ -0,0 +1 @@
+TextRecogniser
```

Comment:
I would suggest you to delete the .idea folder and add it to the `.gitignore` file so that anyone who clones the repo doesn't face any issue in building.

Sometimes configuration files inside the `.idea` folder overrides, it is best to remove them and let the cloner decide which build configurations to use.

Reply:
I didnt exactly get what you meant by adding the .idea folder to the .gitignore file. 
for now i have just removed the .idea from the project, google says that its safe to delete the .idea file. 
but let me know if there is any way i can add the .idea folder to the .gitignore.

Reply:
also will this issue be considered for hacktoberfest😅
Im fine even if its not considered for the fest (as i got to learn few things) i was jst asking coz i saw some confusion regarding the hacktoberfest labels. thanks!

Reply:
@pratik2315 The `.idea` contains local settings for Android studio, like gradle location, jdk build configurations, and many IDE specific files, which can cause issues to someone else if they clone the repo that have the `.idea` folder. I told you to remove it because I myself got that error xD

And yeah, the `.gitignore` file just keeps track of what files and folders to exclude from commuting to the repo, you can learn how to use it from the docs/any tutorial, it's very helpful to keep local configuration files away from being pushed to public repos.